File: pyg_01/pyg_sellergoods_01/pyg_sellergoods_interface_01/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;

public interface BrandService {
    List<TbBrand> findAll();

    PageResult<TbBrand> findPage(int page, int size);

    TbBrand findOne(long id);

    void save(TbBrand brand);

    void delete(String ids);
}

File: pyg_01/pyg_manager_web_01/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    //查询全部
    this.findAll=function () {
        return $http.get('../brand/findAll');
    };
    //保存,参数是页面数据封装的对象
    this.save=function (entity) {
        return $http.post('../brand/save',entity);
    };
    //分页查询
    this.findPage=function (page,size) {
        return $http.get('../brand/findPage/'+page+"/"+size);
    };
    //根据id查询
    this.findOne=function (id) {
        return $http.get('../brand/findOne/'+id);
    };
    //根据选中的id删除
    this.delete=function (selectIds) {
        return $http.get('../brand/delete/'+selectIds);
    }
});